packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -43,6 +44,7 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
 
     constructor(
         private readonly tokenProvider: ITokenProvider,
+        private readonly urlResolver: IUrlResolver,
```

Comment:
this can't take any IUrlResolver. it should explicitly be the RouterliciousUrlResolver

Reply:
What's the reason? How's it different from `ITokenProvider` parameter?

Reply:
can i pass and odsp urls resovler here? will that work

Reply:
i really think this should just internally reuse any code it needs.

Reply:
The idea is yes, any url resolver that supports the new create flow should be able to do that

Reply:
that doesn't sound right to me. this new create flow is FRS specific, and very different from odsp. the odsp resolved won't and shouldn't be expected to work here